var/spack/repos/builtin/packages/tassel/package.py

```diff
@@ -13,7 +13,8 @@ class Tassel(Package):
     homepage = "https://www.maizegenetics.net/tassel"
     git      = "https://bitbucket.org/tasseladmin/tassel-5-standalone.git"
 
-    version('2017-07-22', commit='ae96ae75c3c9a9e8026140b6c775fa4685bdf531')
+    version('3.0.174', commit='612a92c0f677d6ec040c17e146671974a3cdc2da', git='git://git.code.sf.net/p/tassel/tassel3-standalone.git')
+    version('5.2.39', commit='ae96ae75c3c9a9e8026140b6c775fa4685bdf531')
```

Comment:
Can you move this version declaration at the top (so that versions are in descending order)?

Reply:
Done.